kore/src/Kore/Attribute/Subsort.hs

```diff
@@ -6,23 +6,41 @@ License     : NCSA
 -}
 module Kore.Attribute.Subsort where
 
+import Control.DeepSeq
+    ( NFData
+    )
 import Data.Hashable
     ( Hashable
     )
+import qualified Generics.SOP as SOP
+import qualified GHC.Generics as GHC
 
 import Kore.Attribute.Parser as Parser
+import Kore.Debug
 
 {- | The @subsort@ attribute. -}
 data Subsort = Subsort
     { subsort :: Sort
     , supersort :: Sort
     }
-    deriving (Eq, Generic, Ord, Show)
+    deriving (Eq, GHC.Generic, Ord, Show)
+
+instance SOP.Generic Subsort
```

Comment:
The style guide specifies one blank line between top-level declarations. (Should we update the guide?)

Reply:
I don't mind leaving a blank line. It's just that I don't know the guide by heart and I saw some similar instances not separated...
Maybe it's good to have them separated; it's harder to confuse them for a single block.